client/src/hooks/datafiles/mutations/useMove.ts

```diff
@@ -0,0 +1,128 @@
+  api,
+  scheme,
+  system,
+  path,
+  destSystem,
+  destPath,
+}: {
+  api: string;
+  scheme: string;
+  system: string;
+  path: string;
+  destSystem: string;
+  destPath: string;
+}): Promise<{ name: string; path: string }> {
+  const body = {
+    dest_system: destSystem,
+    dest_path: destPath,
+  };
+  const url = `/api/datafiles/${api}/move/${scheme}/${system}/${path}/`;
+  const request = await apiClient.put(url, body);
+  return request.data;
+}
+
+function useMove() {
+  const dispatch = useDispatch();
+  const { selectedFiles: selected } = useSelectedFiles();
+  const status = useSelector(
+    (state: any) => state.files.operationStatus.move,
+    shallowEqual
+  );
+
+  const { api, scheme } = useSelector(
+    (state: any) => state.files.params.FilesListing
+  );
+  const setStatus = () => {
+    dispatch({
+      type: 'DATA_FILES_SET_OPERATION_STATUS',
+      payload: { operation: 'move', status: 'RUNNING' },
+    });
+  };
+
+  const { mutateAsync } = useMutation({
+    mutationFn: moveFileUtil,
+  });
+
+  const move = ({
+    destSystem,
+    destPath,
+    callback,
+  }: {
+    destSystem: string;
+    destPath: string;
+    callback: () => void;
+  }) => {
+    const filteredSelected = selected.filter(
+      (f: any) => status[f.id] !== 'SUCCESS'
+    );
+    const moveCalls: Promise<any>[] = filteredSelected.forEach((file: any) => {
+      dispatch({
+        type: 'DATA_FILES_SET_OPERATION_STATUS_BY_KEY',
+        payload: {
+          status: 'RUNNING',
+          key: file.id,
+          operation: 'move',
+        },
+      });
+      return mutateAsync(
+        {
+          api: api,
+          scheme: scheme,
+          system: file.system,
+          path: file.path,
+          destSystem: destSystem,
+          destPath: destPath,
+        },
+        {
+          onSuccess: () => {
+            dispatch({
+              type: 'DATA_FILES_SET_OPERATION_STATUS_BY_KEY',
+              payload: {
+                status: 'SUCCESS',
+                key: (index: string) => index,
+                operation: 'move',
+              },
+            });
+            dispatch({
+              type: 'DATA_FILES_TOGGLE_MODAL',
+              payload: { operation: 'move', props: {} },
+            });
+            dispatch({
+              type: 'ADD_TOAST',
+              payload: {
+                message: `${
+                  filteredSelected.length > 1
+                    ? `${filteredSelected.length} files`
+                    : 'File'
+                } moved to ${truncateMiddle(destPath, 20) || '/'}`,
+              },
+            });
+            callback();
+          },
+          onError: () => {
+            dispatch({
+              type: 'DATA_FILES_SET_OPERATION_STATUS_BY_KEY',
+              payload: {
+                status: 'ERROR',
+                key: (index: string) => index,
+                operation: 'move',
+              },
+            });
+            console.log('File error');
```

Comment:
Can remove this console log

Reply:
Thanks for catching this! I just removed it.